### PR TITLE
Update pipelines.adoc

### LINKED ISSUE #18: Terminology changes for Kabanero 0.6
Per docs issue 267

### PR DESCRIPTION
**Pull request by jantley-ibm**
closes #18

### CODE REVIEW

File: pipelines.adoc [hunk 1]
```diff
@@ -251,12 +251,12 @@ If the pipeline run was successful, you can see a Docker image in our Docker reg
...
 
-To find solutions for common issues and troubleshoot problems with pipelines, see the link:https://github.com/kabanero-io/kabanero-pipelines/blob/master/Troubleshooting.md[Kabanero Pipelines Troubleshooting Guide].
+To find solutions for common issues and troubleshoot problems with pipelines, see the link:https://github.com/kabanero-io/kabanero-pipelines/blob/master/Troubleshooting.md[Pipelines Troubleshooting Guide].
 
 === Related links
 
-- link:https://www.youtube.com/watch?v=MfS05SU9yIM&feature=youtu.be[Kabanero pipelines demo video part one]
-- link:https://www.youtube.com/watch?v=643sJczM9bU&feature=youtu.be[Kabanero pipelines demo video part two]
-- link:https://www.youtube.com/watch?v=M-5VqXtKyqw&feature=youtu.be[Kabanero pipelines demo video part three]
+- link:https://www.youtube.com/watch?v=MfS05SU9yIM&feature=youtu.be[Pipelines demo video part one]
```
Comment (tseelbach) [inline pipelines.adoc]: The videos are now obsolete

Comment (jantley-ibm) [reply pipelines.adoc]: removed

File: pipelines.adoc [hunk 2]
```diff
@@ -4,26 +4,26 @@ permalink: /guides/pipelines-getting-started/
...
+If you are building a new application stack, the default tasks and pipelines are automatically pulled into your application stack hub repo. If you want to customize the default tasks or pipelines, you can choose to apply your changes either to all application stacks or to a specific application stack.  To apply your changes to all,  update the files in the `incubator/common/pipelines/default` directory in your application stack hub repo. If you want to update tasks or pipelines for one, make your changes in the pipelines directory under that application stack in your application stack hub repo.
 
 === The build, push and deploy pipeline
 
 https://github.com/kabanero-io/collections/blob/master/incubator/common/pipelines/default/build-push-deploy-pipeline.yaml[Build-push-deploy-pipeline.yaml]
```
Comment (kvijai82) [inline pipelines.adoc]: Please update to: https://github.com/kabanero-io/collections/blob/master/incubator/common/pipelines/default/build-deploy-pl.yaml[build-deploy-pl.yaml]

File: pipelines.adoc [hunk 3]
```diff
@@ -221,16 +221,16 @@ vi default-collection-image-registry-secret.yaml
 oc apply -f default-collection-image-registry-secret.yaml
```
Comment (kvijai82) [inline pipelines.adoc]: Please update to oc apply -f default-stack-image-registry-secret.yaml

File: pipelines.adoc [hunk 4]
```diff
@@ -161,19 +161,19 @@ oc apply -f <pipeline.yaml>
...
-The following steps explain how to run pipelines against custom built collection images instead of the provided Kabanero collections.
+The following steps explain how to run pipelines against custom built application stack images instead of the provided application stacks.
 
-=== Set up a container registry URL for the custom collection image
+=== Set up a container registry URL for the custom application stack image
 
-By default, pipelines pull the collection images for Docker hub. If you are publishing your collection images to any other repository, use the following process to configure the custom repository from which your pipelines pull the container images.
+By default, pipelines pull the application stack images for Docker hub. If you are publishing your application stack images to any other repository, use the following process to configure the custom repository from which your pipelines pull the container images.
 
 . After you clone the `kabanero-pipelines` repository, find the `collection-image-registry-map.yaml` configmap template file. Add your container registry URL to this file in place of the `default-collection-image-registry-url` statement.
```
Comment (kvijai82) [inline pipelines.adoc]: collection-image-registry-map.yaml should be stack-image-registry-map.yaml (https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/sample-helper-files/stack-image-registry-map.yaml)

File: pipelines.adoc (hunk 3 above)
Comment (jantley-ibm) [reply pipelines.adoc]: fixed, along with several other instances nearby

File: pipelines.adoc (hunk 2 above)
Comment (jantley-ibm) [reply pipelines.adoc]: fixed

File: pipelines.adoc (hunk 4 above)
Comment (jantley-ibm) [reply pipelines.adoc]: fixed, along with several other instances nearby

**Review by kvijai82 — APPROVED** [review]

**Pull request merged**